tests/test_nginx_service.py

```diff
@@ -1,3 +1,4 @@
+from lib2to3.pgen2.token import EQUAL
```

Comment:
This does not appear to be used... what even is this? Never seen it before, looks odd.